install/kubernetes/helm/istio/charts/ingress-gateway/templates/clusterrole.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.global.rbacEnabled }}
```

Comment:
How about keeping the ingress as is (it is a gateway) ?

Reply:
Yup, we want this as a separate thing, but I don't think it should require any permissions (it should just talk to Pilot, right?).